.github/workflows/ci-ubuntu-18-nightly.yml

```diff
@@ -39,4 +39,36 @@ jobs:
         # this is needed to create network namespaces for the ebpf tests.
       - name: Run tests (Ubuntu 18.04)
         run: |
-          sudo -E docker run --privileged -w /p4c/build -e $CTEST_PARALLEL_LEVEL p4c ctest --output-on-failure --schedule-random
+          sudo -E docker run --privileged -w /p4c/build -e $CTEST_PARALLEL_LEVEL p4c ctest --output-on-failure --schedule-random \
+          -E "p4tc_samples_stf|p4tc_cleanup|p4tc_setup"
+  # Run p4tc stf tests
+  test-ubuntu18-p4tc-stf:
```

Comment:
I would consider putting this in a separate file. You could also run it nightly, like the ubuntu-18 tests.

Reply:
Moved it to a separate file ("ci-ubuntu-18-nightly-p4tc-stf.yml")